### PR TITLE
Revert "removes the incusion implant (#9090)"

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

This reverts commit 8a072c686646d7c963eb63501172887a5b4b308a of PR #9090, adding back the Incursion radio implant

## Why It's Good For The Game

This PR was merged within 12 hours with zero discussion or review, and I don't think time was taken to consider the repercussions of doing so.

- Incursions now rely entirely on finding each other in person / with tracking, forcing communication with radio frequencies, or using insecure PDA messaging which could easily be identified as suspicious by security.
- Incursionists cannot purchase syndicate keys.
- Incursions still receive only 15TC despite losing at least 3TC in value.
- Incursionists don't receive codewords, and even if they did they would be useless since they're only used to identify other agents, and they already know who the other incursionists are.

Incursions while powerful vary heavily in their ability to get things done. Meeting each other in person is also strictly advantageous if that's what we're trying to accomplish. Many incursions establish a hideout. However, having a way to communicate while apart is VITAL to an incursion's function, otherwise you just get huge teamwork blunders, especially as many players are not willing to stick around and elaborate on a plan. If you need help in the moment, there is NO way to call for it. Being arrested? Tough luck. This forces incursions into stealth and they cannot rely on teammates in combat.

In conclusion, having the comms allows incursions to avoid using as much stealth, driving round conflict, and better locate members who are worse at communicating or sticking with the group. It also allows on-the-fly coordination that is not possible otherwise, as they cannot purchase radio keys anyway.

Antagonists being powerful gives sec a worthy challenge - without effective communications an incursion is no better than a collection of random traitors with less TC, and sharing objectives. This is supposed to be a tight-knit team who have infiltrated the station, a real threat to security. While it is true that incursions could reasonably stomp sec and take over, a radio uplink gives them little extra combat advantage other than organizing. They already have plenty of TC to gain effective murder devices should they wish to eliminate sec.

## Testing Photographs and Procedure

N/A

## Changelog
:cl:
balance: Re-added incursion's radio keys.
/:cl:

### CODE REVIEW

**Comment by CydiaLamiales** [comment]
The implant also allowed incursion to coordinate attacks too well - with their special TC deals and guaranteed teammates they often times completelly rolled over all of Security and Command.

I do think the removal of the **guaranteed nearly undetectable way to communicate with the entirety of your team** is good, as it actually forces you to plan things out and find alternative ways to communicate instead of bumrushing your objectives.

It wasn't a 'reasonable' challenge for security, it was a one-sided stomp.

**Comment by itsmeow** [comment]
> The implant also allowed incursion to coordinate attacks too well

Removing it makes coordinating attacks unreasonably hard. The entire fun of an incursion is the coordination. Both measures are too drastic in each direction.

Any good compromise you can think of? We want incursions to have minimum communication roundstart and have potential to improve communication, while not imposing weird arbitrary restrictions. Having the ability to coordinate is essential to an incursion, especially if they are not robust.

An incursion's core is in numbers, not strength. We could reduce their TC further, or reduce member counts, or increase security requirements... any of the above, if it is true that incursions are always stomping sec. I don't think removing the radio key is a good method though.

If we added a guaranteed discount for it, potentially? Or maybe some kind of cooldown?

**Comment by Fronsis** [comment]
> This PR was merged within 12 hours with zero discussion or review, and I don't think time was taken to consider the repercussions of doing so.

Did you bother to read the discussion we had on both of these threads? 
photo

> Incursions now rely entirely on finding each other in person / with tracking, forcing communication with radio frequencies, or using insecure PDA messaging which could easily be identified as suspicious by security.

That's perfect, it encourages roleplay and properly talking with your teammates rather than talk with them across the station coordinating massive attacks with zero drawbacks and at the same time having access to all of the other radio channels, which means you also know every station move, what's wrong with being identified as a suspect? Nothing guarantees you will, you just need to play your cards properly, if you go into cabin 4 for twenty minutes with your five teammates it's obviously going to be suspicious, now you need to have a better coordination

> Incursionists cannot purchase syndicate keys.

They should have another method, i agree but one that doesn't collide with the ex-comms and is not absolutely busted as explained in the discussion channels on discord


> Incursions still receive only 15TC despite losing at least 3TC in value.

They can overwhelm security and command easily by purchasing crates or spending wisely their TC, specially since it can roll up to SIX members, it's about coordination and teamwork rather than everyone going to do their own, they absolutely do **not** need more TC

But yeah as Wilson said above me.. it was 90% of the time a stomp, hell i've observed a few rounds ever since the change got made and they still have a positive winrate from what i saw, and considering the state Security is at the moment, they're in very much of need of an extra breath.

Also


> otherwise you just get huge teamwork blunders, especially as many players are not willing to stick around and elaborate on a plan. If you need help in the moment, there is NO way to call for it. Being arrested? Tough luck. This forces incursions into stealth and they cannot rely on teammates in combat.

Well, the same way Security doesn't get the best of the teamwork sometimes it makes sense for the antagonists to also have some slips, if those players are not willing to stick around and work as a team then they will start learning once those oopsies lead to them being arrested, killed, etc

It's also important because it makes the other players value the positions of the others and maybe a need to keep a check on them from time to time, there's no.. difficulty on someone saying ; All good?
; x3 yes
; sec got me help (the second they lost the fight, and if they play smart they can just sneakily report sec positions while on the cell or on perma without zero drawbacks)

Basically it's just too powerful due to the combination of possibilities they have, they don't need it, they can work around it

I do agree they should have something similar but not as strong, the current one gives too much of a headstart alongside making space for excomm cheese and other stuff **mentioned in the threads** hence why Bacon proposal was intersting.

**Comment by itsmeow** [comment]
> mentioned in the threads

See, I did not see the threads nor get any attempt to put thoughts into it. I am more upset that a large balance PR like this one was merged within 12hrs with no chance for other maintainers to review it, even if that discussion took place, it wasn't visible on the PR which just made me think it was a speedmerge.

I think more discussion is needed on how to nerf incursion, I agree weakening their roundstart communication is important but currently it throws a huge rock in incursion and wasn't thought out enough imo.

**Comment by itsmeow** [comment]
What if they had a limited recharge mind-link type deal?

They could send large messages in chat to each other once every 120s or so, that way they can coordinate on big ideas (meet at X location, bomb the AI sat, help with sec at X location) but can't just parrot every detail to each other.

**Review by PowerfulBacon — APPROVED** [review]
While I agree with the nerf and think that no comms is better than comms, the merge was poorly handled, should have been left at least 24h as per policy and should have been requested to give them a meeting point.
